tests/crm/test_crm.py

```diff
@@ -69,6 +69,17 @@ def ignore_expected_loganalyzer_exceptions(enum_rand_one_per_hwsku_frontend_host
         loganalyzer[enum_rand_one_per_hwsku_frontend_hostname].ignore_regex.extend(ignoreRegex)
 
 
+@pytest.fixture(scope="module")
```

Comment:
instead of module scope, we can make it function since it is only needed by this testcase